Improve CedarScript and code formatting

kj7rrv opened the pull request:
## Improve CedarScript definitions code

I made some changes to simplify maintenance, and added `log_good`, `log_spam`,
`log_probably_good`, and `log_probably_spam` as actions, for when we add
auto-classification. These are not supported by CedarSentinel yet, so they
don't do anything, but the CedarScript interpreter can handle them without
errors now.

## Reformat code

I just ran `black *.py cedarscript/*.py`.

kj7rrv committed:
Improve CedarScript definitions code

* Improve code style
* Add log_good, log_spam, log_probably_good, and log_probably_spam

kj7rrv committed:
Reformat code

fire219 reviewed (changes requested):
The CedarScript improvements are great to see, but the code reformatting IMO makes readability worse in more cases than it improves it.

kj7rrv committed:
Revert "Reformat code"

This reverts commit 03c19c36f88de66b4487cf3bb71f782256dd8f91.

kj7rrv committed:
Reformat code (again)

Black without line length limits (I couldn't find a way to disable the
limit, so I set it to 10,000 characters)
